src/ucp/rndv/rndv.c

```diff
@@ -44,18 +45,22 @@ static int ucp_rndv_is_recv_pipeline_needed(ucp_request_t *rndv_req,
     uint64_t mem_types;
     int i;
 
+    /* TODO: add logic to find actual fragment type needed later */
+    *frag_mem_type  = UCS_MEMORY_TYPE_HOST;
```

Comment:
should we extend the logic here?

Reply:
Need to address this once we determine how to choose the right memory type for the pipeline (the prototype uses this logic https://github.com/openucx/ucx/pull/6911/files#diff-8521f6163537ad4bda8f249fbb338f3ef8de6728185962c78ad7231b5f97c26dR126-R161)